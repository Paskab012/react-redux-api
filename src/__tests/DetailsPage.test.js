import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/ConfigureStrore';
import Details from '../components/details/DetailsPage';

describe('Detail page component test', () => {
  it('Should return/render the item detail', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <Details />
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
