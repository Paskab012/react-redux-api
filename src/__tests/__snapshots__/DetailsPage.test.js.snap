// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail page component test Should return/render the item detail 1`] = `
<div
  className="text-center"
>
  <div>
    Loading, wait a moment...
  </div>
</div>
`;
